specs/wp-signup-spec.js

```diff
@@ -168,14 +173,48 @@ testDescribe( `[${host}] Sign Up  (${screenSize}, ${locale})`, function() {
 										} );
 									} );
 
-									test.describe( 'Step Eight: Can activate my account from an email', function() {
-										test.before( function() {
-											return this.emailClient = new EmailClient( signupInboxId );
+									test.describe( 'Step Eight: Can not publish until email is confirmed', function() {
+										test.it( 'Can see a disabled publish button', function() {
+											const blogPostTitle = dataHelper.randomPhrase();
+											const blogPostQuote = dataHelper.randomPhrase();
+
+											this.adminTopBar = new AdminTopBar( this.driver );
+											this.adminTopBar.createNewPost();
+
+											this.editor = new EditorPage( this.driver );
+											this.editor.enterTitle( blogPostTitle );
+											this.editor.enterContent( blogPostQuote + '\n' );
+											return this.editor.publishEnabled().then( ( enabled ) => {
+												return assert.equal( enabled, false, 'Publish button is not disabled when activation link has not been clicked' );
+											} );
```

Comment:
I like that idea. It doesn't hurt to have that additional step to make sure the message shows up.